docs/build-current

```diff
@@ -8,5 +8,7 @@ for script in scripts/* ; do
   "$script" > "src/generated/${output%.*}.html"
 done
 
+python3 render_examples > src/content/addons-examples.md
```

Comment:
This should probably live in scripts/ so that it is covered by lines 5-9.

Reply:
I played around with this a bit - I couldn't get it to work. Maybe your Hugo knowledge can solve it: I'm generating a markdown file from my python script. Hugo then does the usual of transforming markdown into html, while running the shortcut functions / templating inline.

If I would make use of the scripts loop, then I would only get a html, which breaks the whole pipeline. Ideas?